pkg/agent/openflow/pipeline.go

```diff
@@ -2579,6 +2579,18 @@ func pipelineClassifyFlow(cookieID uint64, protocol binding.Protocol, pipeline b
 		Done()
 }
 
+// igmpEgressFlow generates flows to match IGMP report to jump to table MulticastRoutingTable.
+// This is because normal multicast egress rule can match IGMP v1 report, when there is egress
+// rule to block multicast traffic, IGMP v1 report will also be blocked, which is not expected.
+func (f *featureMulticast) igmpEgressFlow() binding.Flow {
```

Comment:
Does it make one flow created by `multicastSkipIGMPMetricFlows` pointless? Are they kind of duplicate?

Reply:
igmpEgressFlow only skips IGMP report, this is because MulticastEgressRuleTable can not distinguish IGMP report from nomarl multicast egress traffic, it makes egress rule does not work as expected.
multicastSkipIGMPMetricFlows skips both IGMP report and IGMP query, multicast stats does not care about IGMP packets and skips all IGMP packets